src/Microsoft.DotNet.ServiceFabric.ServiceHost/DelegatedStatefulService.cs

```diff
@@ -73,7 +65,7 @@ protected override IEnumerable<ServiceReplicaListener> CreateServiceReplicaListe
             }
 
             Type[] interfaces = typeof(TServiceImplementation).GetAllInterfaces()
-                .Where(iface => iface.IsAssignableTo<IService>())
```

Comment:
>IsAssignableTo [](start = 38, length = 14)

I always wondered where this very nice IsAssignableTo method was coming from. I guess autofac was giving us this goodness.

Reply:
Yeah, I was surprised it vanished too.